models/spatial/referencedata.yaml

```diff
@@ -0,0 +1,55 @@
+id: https://www.biorxiv.org/content/10.1101/2023.01.22.525049v1
+name: referencedata
+prefixes:
+  linkml: https://w3id.org/linkml/
+imports:
+  - linkml:types
+default_range: string
+
+classes:
+
+  Subject:
+    attributes:
+      identifier:
+        required: true
+      species:
+        required: true
+      biological_sex:
+#        range: sex_enum
+        required: true
+      method_information:
+        required: true
+      processing_information:
+        required: true
+
+  Image:
+    attributes:
```

Comment:
this could benefit from the considerations in bids. see the bids microscopy paper, but also consider the bids standard for references to coordinate systems.